### PR TITLE
Task1 part3 responsive design

### PR DESCRIPTION
**Pull request by Mohsen-Awad**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (araraltawarah98) [inline .vscode/settings.json:3]: add the end of the file line

File: task_1/photo/menu-icon.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0z" fill="none"/><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"/></svg>
```
Comment (araraltawarah98) [inline task_1/photo/menu-icon.svg:1]: add the end of the file line

**Review by araraltawarah98 — CHANGES_REQUESTED** [review]

**Review by abood454 — APPROVED** [review]

**Review by araraltawarah98 — APPROVED** [review]

File: task_1/index.html [hunk 3]
```diff
@@ -13,17 +13,20 @@
...
-                <div id="about-item" class="header-item" onclick="onMenuItemClick('about-item')">About Us</div>
-                <div id="lands-item" class="header-item" onclick="onMenuItemClick('lands-item')">Our Lands</div>
-                <div id="suppliers-item" class="header-item" onclick="onMenuItemClick('suppliers-item')">Our Suppliers</div>
-                <div id="products-item" class="header-item" onclick="onMenuItemClick('products-item')">Products</div>
-                <div id="contact-item" class="header-item" onclick="onMenuItemClick('contact-item')">Contact Us</div>
-            </div>
-            <div class="header-images">
-                <div><img class="shopping-image" src="./photo/shopping-cart.png" alt="shopping cart"></div>
-                <div><img class="profile-image" src="./photo/profile-image.png" alt="profile image"></div>
+            <div class="navbar" id="navbar">
```
Comment (mmasaid) [inline task_1/index.html:16]:
```suggestion
            <div class="navbar" id="menu-navbar">
```

File: task_1/index.html [hunk 4]
```diff
@@ -136,5 +139,7 @@ <h4>Double Espresso</h4>
         </div>
     </div>
     <script src="index.js"></script>
+
+    <table id="employees"></table>
```
Comment (mmasaid) [inline task_1/index.html:143]: git ride this line no need for the table 
I think it was by mistake 
```suggestion
```

File: task_1/index.js [hunk 5]
```diff
@@ -1,8 +1,27 @@
-const onMenuItemClick = (elementid) => {
+const onHeaderItemClick = (elementid) => {
     let chosen_item = document.getElementById(elementid);
     let header_items = document.getElementsByClassName("header-item");
     for (let i = 0; i < header_items.length; i++)
         header_items[i].classList.remove("active");
 
     chosen_item.classList.add("active");
 }
+
```
Comment (mmasaid) [inline task_1/index.js:9]:
```suggestion
```

File: task_1/index.js [hunk 6]
```diff
@@ -1,8 +1,27 @@
...
+
+const onMenuItemClick = (elementid) => {
+    let chosen_item = document.getElementById(elementid);
+    let menu_items = document.getElementsByClassName("menu-item");
+    for (let i = 0; i < menu_items.length; i++)
```
Comment (mmasaid) [inline task_1/index.js:14]:
```suggestion
    for (let i = 0; i < menu_items.length; i++){
    ...
    }
```

File: task_1/index.js [hunk 7]
```diff
@@ -1,8 +1,27 @@
...
+        menu_items[i].classList.remove("active");
+
+    chosen_item.classList.add("active");
+}
+
```
Comment (mmasaid) [inline task_1/index.js:19]:
```suggestion
```

File: task_1/index.js [hunk 8]
```diff
@@ -1,8 +1,27 @@
...
+
+document.getElementById("menu-icon").addEventListener("click", () => {
+    let menu_item = document.getElementById("navbar");
```
Comment (mmasaid) [inline task_1/index.js:22]:
```suggestion
    let menu_item = document.getElementById("menu-navbar");  
```

File: task_1/style.css [hunk 9]
```diff
@@ -32,7 +30,7 @@
 .header-images {
     align-items: center;
     display: flex;
-    justify-content: space-between;
+    justify-content: end;
```
Comment (mmasaid) [inline task_1/style.css:33]:
```suggestion
    justify-content: flex-end;
```

File: task_1/style.css [hunk 10]
```diff
@@ -153,3 +151,66 @@
...
     font-weight: bolder;
 }
+
+.menu-icon{
+    width: 50px;
+    height: 50px;
+    cursor: pointer;
+    display: none;
+    margin-top: 20px;
+}
```
Comment (mmasaid) [inline task_1/style.css:161]:
```suggestion
Alphabetize please 
.menu-icon{
    cursor: pointer;
    width: 50px;
    height: 50px;
    display: none;
    margin-top: 20px;
}
```

File: task_1/style.css [hunk 11]
```diff
@@ -153,3 +151,66 @@
...
+    }
+    .navbar {
+        flex-direction: column;
+        visibility: hidden;
+        justify-content: space-between;
+        background-color:rgb(167, 184, 216);
+        z-index: 1;
+        margin-top: 370px;
+        width: 100%;
+        justify-content: flex-end;
```
Comment (mmasaid) [inline task_1/style.css:202]: Alphabetize please 
```suggestion
......
        flex-direction: column;
        visibility: hidden;
        justify-content: space-between;
        background-color:rgb(167, 184, 216);
        z-index: 1;
        margin-top: 370px;
        width: 100%;
        justify-content: flex-end;
```

File: task_1/style.css [hunk 12]
```diff
@@ -153,3 +151,66 @@
...
+    .header-item{
+        margin: 10px;
+    }
+    .menu-icon{
+        display: inline;
+    }
+    .sub-title-p{
+        width: auto;
+    }
+    .header{
```
Comment (mmasaid) [inline task_1/style.css:213]: line space between class 
Apply it in the whole file
```suggestion

    .header{
```

**Pull request merged**